File: 백준/Bronze/2869. 달팽이는 올라가고 싶다/달팽이는 올라가고 싶다.cc
#include <iostream>
using namespace std;

int main() {
    long long A, B, V;
    cin >> A >> B >> V;

    long long day = (V - A + (A - B) - 1) / (A - B) + 1;
    cout << day << '\n';

    return 0;
}

File: 백준/Bronze/2869. 달팽이는 올라가고 싶다/달팽이는 올라가고 싶다.swift
import Foundation

let input = readLine()!.split(separator: " ").map { Int64($0)! }
let A = input[0]
let B = input[1]
let V = input[2]

let day = (V - A + (A - B) - 1) / (A - B) + 1
print(day)